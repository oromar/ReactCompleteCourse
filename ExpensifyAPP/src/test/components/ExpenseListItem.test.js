import React from 'react'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'
import ExpenseListItem from '../../components/expense-list-item'
import expenses from '../fixtures/expenses'

test('should render expense list item correctly', () => {
    const wrapper = shallow(<ExpenseListItem {...expenses[0]} />)
    expect(toJson(wrapper)).toMatchSnapshot()
})