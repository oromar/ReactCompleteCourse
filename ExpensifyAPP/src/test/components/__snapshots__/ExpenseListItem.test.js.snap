// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense list item correctly 1`] = `
<div>
  <h3>
    <NavLink
      activeClassName="active"
      aria-current="page"
      to="/edit/1"
    >
      rent
    </NavLink>
  </h3>
  <p>
    100
     - 
    Dec 17th 1995
  </p>
</div>
`;
